src/main/java/aether/Aether.java

```diff
@@ -0,0 +1,206 @@
+package aether;
+
+import aether.task.Task;
+import aether.task.Todo;
+import aether.task.Deadline;
+import aether.task.Event;
+import aether.ui.Display;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Aether class represents the main logic for a simple task manager program.
+ * It allows users to add, delete, mark, unmark tasks, and display tasks.
+ */
+public class Aether {
```

Comment:
I know it's a work in progress, but remember to add more comments for this class.